service/src/main/java/bio/terra/externalcreds/services/StatusServiceCache.java

```diff
@@ -0,0 +1,81 @@
+package bio.terra.externalcreds.services;
+
+import bio.terra.externalcreds.SamStatusDAO;
+import bio.terra.externalcreds.generated.model.Provider;
+import bio.terra.externalcreds.generated.model.SubsystemStatusDetail;
+import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+/**
+ * Creating a provider client requires an api call to [provider
+ * issuer]/.well-known/openid-configuration. That information almost never changes and we don't want
+ * to hammer that api. Therefore this cache.
+ *
+ * <p>The cache is reset every 6 hours to detect infrequent changes.
```

Comment:
I'm wondering if 6 hours is too long for caching Sam's status, since the ECM status relies on Sam being up 🤔

Reply:
Oh I realized this was copied over from `ProviderOAuthClientCache` and it's actually set to 1 minute here for Sam and 5 minutes for the providers. Could you update this comment?

Reply:
Also if the provider cache is reset every 6 hours, maybe checking it every 5 minutes is too often?